Makefile

```diff
@@ -25,6 +27,14 @@ clean:
 	helm delete orkestra -n orkestra 2>&1
 	kind delete cluster --name orkestra 2>&1
 
+# Build the docker image and push it to local docker registry
+local-build:
+	docker build . -t localhost:5000/orkestra:dev
+
+# Update the Orkestra deployment with `localhost:5000/orkestra:dev`
+local-deploy:
+	helm upgrade orkestra chart/orkestra -n orkestra --create-namespace --set image.repository=localhost:5000/orkestra --set image.tag=dev
```

Comment:
This fails on a KinD cluster because there is not local-registry available inside the kind node.
```yaml
│   containerStatuses:                                                                                                                                                                                         │
│   - image: localhost:5000/orkestra:dev                                                                                                                                                                       │
│     imageID: ""                                                                                                                                                                                              │
│     lastState: {}                                                                                                                                                                                            │
│     name: orkestra                                                                                                                                                                                           │
│     ready: false                                                                                                                                                                                             │
│     restartCount: 0                                                                                                                                                                                          │
│     started: false                                                                                                                                                                                           │
│     state:                                                                                                                                                                                                   │
│       waiting:                                                                                                                                                                                               │
│         message: Back-off pulling image "localhost:5000/orkestra:dev"                                                                                                                                        │
│         reason: ImagePullBackOff
```